### PR TITLE
Add link to pixijselementals.com to "Learn" section in main page of PixiJS readme

### PR DESCRIPTION
**Pull request by domis86**
##### Description of change

Add link to @miltoncandelero tutorial https://www.pixijselementals.com/ to "Learn" section in main page of PixiJS README.md

Because is very good tutorial :)

Currently is using pixi.js version 6.3.0 - so quite recent 

##### Pre-Merge Checklist

- [x] Documentation is changed or added

### CODE REVIEW

**Review by bigtimebuddy — APPROVED** [review]
100% support this idea. @miltoncandelero's guides are awesome and we should call them out here. I'd ask @miltoncandelero to weigh-in on how he wants this described. And I also, don't have a problem listing this first.

File: README.md [hunk 1]
```diff
@@ -33,7 +33,9 @@ PixiJS has full [WebGL](https://en.wikipedia.org/wiki/WebGL) support and seamles
 
 ### Learn ###
 - Website: Find out more about PixiJS on the [official website](https://www.pixijs.com/).
-- Getting started: Check out @kittykatattack's comprehensive [tutorial](https://github.com/kittykatattack/learningPixi).
+- Getting started:
+    - Check out @kittykatattack's comprehensive [tutorial](https://github.com/kittykatattack/learningPixi).
+    - also check out @miltoncandelero tutorial with PixiJS recipes, best practices and typescript setup [here](https://www.pixijselementals.com/)
```
Comment (bigtimebuddy) [inline README.md:38]: Small punctuation changes
```suggestion
    - Also check out @miltoncandelero's tutorials with PixiJS recipes, best practices and TypeScript setup [here](https://www.pixijselementals.com/)
```

**Comment by miltoncandelero** [comment]
> I'd ask @miltoncandelero to weigh-in on how he wants this described. And I also, don't have a problem listing this first.

I have no problem with the description given nor with appearing first 😅

My guides and recipes are aimed toward videogames and use npm, typescript and webpack.

Maybe mentioning that the guide is based on npm and typescript is worth so people won't go in expecting an umd `PIXI.` guide 🤔

File: README.md [hunk 2]
```diff
@@ -33,7 +33,9 @@ PixiJS has full [WebGL](https://en.wikipedia.org/wiki/WebGL) support and seamles
 
 ### Learn ###
 - Website: Find out more about PixiJS on the [official website](https://www.pixijs.com/).
-- Getting started: Check out @kittykatattack's comprehensive [tutorial](https://github.com/kittykatattack/learningPixi).
+- Getting started:
+    - Check out @kittykatattack's comprehensive [tutorial](https://github.com/kittykatattack/learningPixi).
+    - Also check out @miltoncandelero's tutorials with PixiJS recipes, best practices and TypeScript setup [here](https://www.pixijselementals.com/)
```
Comment (domis86) [inline README.md:38]:
```suggestion
    - Also check out @miltoncandelero's PixiJS tutorials aimed toward videogames with recipes, best practices and TypeScript / npm / webpack setup [here](https://www.pixijselementals.com/)
```

Comment (domis86) [reply README.md:38]: @miltoncandelero @bigtimebuddy  WDYT?

Comment (bigtimebuddy) [reply README.md:38]: LGTM

**Comment by bigtimebuddy** [comment]
Thanks @domis86

**Pull request merged**